hlxsites/merative2
# feat(headshot-list-marketo): add Headshot Link block as fragment below body text area - 774 & 859

## Issue

Fixes - https://jira.sdlc.merative.com/browse/MERATIVE-774 and https://jira.sdlc.merative.com/browse/MERATIVE-859

## Description

**New**

- This PR adds a new block called `headshot-list` that acts as a content fragment, querying the `people/query-index.json` and pulling in the metadata needed to display the author's information in the headshot list pattern.

**Changed**

- Clean up and alters the construction of the DOM to account for two columns.


## Content Author Changes
- Authors and contributors for some of the Events/Webinar will need to be moved into the `/people` directory within our SharePoint folder.
- We'll need to adjust the SharePoint doc structure to account for the metadata section for Webinar pages where the Marketo form is located on the left-hand side and other contacts pages where it's displayed on the right column.
- @keith-kaplan @scottbridgeman We'll need to update the existing pages where Marketo is being used so that they're using the proper metadata classes (see below).
- **Regular content body with two columns (Left Col: Text/Body) | (Left Col: Marketo Form)**
  - [SharePoint Draft example](https://merative.sharepoint.com/:w:/r/sites/Merative.comWebsite/_layouts/15/Doc.aspx?sourcedoc=%7B2990610C-2FFD-44F8-B641-1606E7E05D51%7D&file=leverage-real-world-data-in-the-cloud.docx&action=default&mobileredirect=true)
  - Preview: https://feat-headshot-list-774--merative2--proeung.hlx.page/drafts/putra/leverage-real-world-data-in-the-cloud
  - ![Screenshot 2023-08-15 at 5 02 09 PM](https://github.com/hlxsites/merative2/assets/1815714/2a731f3d-4c43-4ca4-9172-161058bc8b7a)

- **Regular content body with two columns (Left Col: Marketo Form) | (Left Col: Text/Body)**
  - [SharePoint example](https://merative.sharepoint.com/:w:/r/sites/Merative.comWebsite/_layouts/15/Doc.aspx?sourcedoc=%7B4DC293D3-BBAF-487F-854A-B57890928537%7D&file=contact.docx&action=default&mobileredirect=true)
  - https://feat-headshot-list-774--merative2--proeung.hlx.page/drafts/putra/contact
  - ![Screenshot 2023-08-15 at 5 01 36 PM](https://github.com/hlxsites/merative2/assets/1815714/fa414e26-305f-4603-918d-7ee204ab21e6)


## Design Specs

- Figma Link - https://www.figma.com/file/FSqQ1LYrVSssEFOZuOnMIE/Final-page-layouts-for-Merative.com?type=design&node-id=1933-47557&mode=design&t=DKh0qauOISJX3DEG-4

## Test URLs
  
- Before (Changes from `main`): https://main--merative2--hlxsites.hlx.page/drafts/putra/leverage-real-world-data-in-the-cloud
- After (Changes from this PR): https://feat-headshot-list-774--merative2--proeung.hlx.page/drafts/putra/leverage-real-world-data-in-the-cloud
  
## Testing Instruction

![Screenshot 2023-08-15 at 5 04 06 PM](https://github.com/hlxsites/merative2/assets/1815714/3c5d1288-5610-4867-83e8-241927ee44ad)

- Visit the new draft page that includes the test content where the author's info is pulled from the `/people` directory. https://feat-headshot-list-774--merative2--proeung.hlx.page/drafts/putra/leverage-real-world-data-in-the-cloud
- Ensure that the page aligns with the design
